docs/user-guide/query/gaffer-syntax/filtering.md

```diff
@@ -340,6 +340,347 @@ properties are returned.
+                .properties("weight")
+                .build())
+            .build();
+        ```
+
+    === "JSON"
+
+        ```json
+        {
+            "view": {
+                "globalElements": [ {
+                    "properties": ["weight"]
+                }]
+            }
+        }
+        ```
+
+    === "Python"
+
+        ```python
+        viewWithGlobalElements = g.View(
+            global_elements=[
+                g.GlobalElementDefinition(
+                    properties=["weight"]
+                ),
+            ],
+            all_edges=False,
+            all_entities=False
+        )
+        ```
+
+Global View definitions can also be used to filter all elements for specific properties.
+
+!!! example ""
+
+    If we want all elements with a `weight` more than 0.4.
+
+    === "Java"
+
+        ```java
+        // Define the View to use
+        final View viewWithGlobalFilter = new View.Builder()
+            .globalElements(new GlobalViewElementDefinition.Builder()
+                .postAggregationFilter(new ElementsFilter.Builder()
+                    .select("weight")
+                    .execute(new IsMoreThan(0.4))
+                    .build())
+                .build())
+            .build();
+        ```
+
+    === "JSON"
+
+        ```json
+        {
+            "view": {
+                "globalElements" : [ {
+                    "postAggregationFilterFunctions" : [ {
+                        "selection" : [ "weight" ],
+                        "predicate" : {
+                            "class" : "IsMoreThan",
+                            "orEqualTo" : false,
+                            "value" : {
+                                "Float" : 0.4
+                            }
+                        }
+                    } ]
+                } ]
+            }
+        }
+        ```
+
+    === "Python"
+
+        ```python
+        viewWithGlobalElements = g.View(
+            global_elements=[
+                g.GlobalElementDefinition(
+                    post_aggregation_filter_functions=[
+                        g.PredicateContext(
+                            selection=["weight"],
+                            predicate=g.IsMoreThan(
+                                value={'java.lang.Float': 0.4},
+                                or_equal_to=False
+                            )
+                        )
+                    ]
+                )
+            ],
+            all_edges=False,
+            all_entities=False
+        )
+        ```
+
+In addition to global definitions, you can add specific view definitions to different element
+groups as you would usually. The global definitions are then merged with each of the element
+specific filters using an AND operator.
+
+!!! example ""
+
+    In this example this would get all `Commit` edges with a `weight` property
+    as well as all `Person` entities with an `age` property.
```

Comment:
would it also get Person entities with a weight property? or is that not how it works

Reply:
Yeah it would apply the weight property to all, I can make that a bit clearer in this comment.